pkg/virtctl/imageupload/imageupload.go

```diff
@@ -152,19 +152,14 @@ func NewImageUploadCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 func usage() string {
 	usage := `  # Upload a local disk image to a newly created DataVolume:
   {{ProgramName}} image-upload dv fedora-dv --size=10Gi --image-path=/images/fedora30.qcow2
-
```

Comment:
Why removing these empty lines? Is this needed for something?

Reply:
Done